services/hooks/src/utils.js

```diff
@@ -0,0 +1,133 @@
+const assert = require('assert').strict;
+const { paginateResults } = require('taskcluster-lib-api');
+
+const lastFireUtils = {
+  // Create a single instance, or undefined, from a set of rows containing zero
+  // or one elements.
+  fromDbRows(rows) {
+    if (rows.length === 1) {
+      return exports.lastFireUtils.fromDb(rows[0]);
+    }
+  },
+  // Create a single instance from a DB row
+  fromDb(row) {
```

Comment:
This is only called from getLastFires, and the result is only used within that handler function.  All it does is change capitalization.  So updating the capitalization in the handler funcs would allow dropping this function.